### PR TITLE
Refactor logging with default log level error

### PR DESCRIPTION
**Pull request by ton31337**
Signed-off-by: Donatas Abraitis <donatas.abraitis@gmail.com>

### CODE REVIEW

File: cmd/nerf/main.go [hunk 1]
```diff
@@ -14,30 +14,52 @@ import (
 
 	"github.com/ton31337/nerf"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 	"google.golang.org/grpc"
 )
 
+func stringToLogLevel(level string) zapcore.Level {
```
Comment (benner) [inline cmd/nerf/main.go:21]: Function missing "verbose" level like before (e.g. debug).

File: cmd/nerf/main.go [hunk 2]
```diff
@@ -14,30 +14,52 @@ import (
...
+}
+
 func main() {
 	server := flag.Bool("server", false, "Start gRPC server to generate config for Nebula")
 	lightHouse := flag.String("lighthouse", "", "Set the lighthouse. E.g.: <NebulaIP>:<PublicIP>")
-	verbose := flag.Bool("verbose", false, "Print verbose output")
+	logLevel := flag.String(
+		"log-level",
+		"error",
+		"Set the logging level - values are 'info', 'warn', and 'error'",
```
Comment (benner) [inline cmd/nerf/main.go]: Same as above

File: cmd/nerf/main.go [hunk 3]
```diff
@@ -82,15 +102,11 @@ func main() {
 		go func() {
 			for range time.Tick(10 * time.Second) {
 				if (time.Now().Unix() - nerf.Cfg.Teams.UpdatedAt) > 3600 {
-					if nerf.Cfg.Verbose {
-						nerf.Cfg.Logger.Info(
-							"Begin-Of-Sync Github Teams with local cache")
-					}
+					nerf.Cfg.Logger.Info(
+						"Begin-Of-Sync Github Teams with local cache")
```
Comment (benner) [inline cmd/nerf/main.go:108]: These should be .Debug() not information (but just it's my opinion)

File: cmd/nerf/main.go [hunk 4]
```diff
@@ -82,15 +102,11 @@ func main() {
...
 					nerf.SyncTeams()
-					if nerf.Cfg.Verbose {
-						nerf.Cfg.Logger.Info(
-							"End-Of-Sync Github Teams with local cache")
-					}
+					nerf.Cfg.Logger.Info(
+						"End-Of-Sync Github Teams with local cache")
```
Comment (benner) [inline cmd/nerf/main.go:111]: Same here

File: cmd/nerf/main.go [hunk 5]
```diff
@@ -118,13 +134,11 @@ func main() {
...
 		}
 
-		if nerf.Cfg.Verbose {
-			nerf.Cfg.Logger.Info("Authorized", zap.String("Login", nerf.Cfg.Login))
-			nerf.Cfg.Logger.Info("Using fastest gRPC endpoint",
-				zap.String("RemoteIP", e.RemoteIP),
-				zap.String("RemoteHost", e.RemoteHost),
-				zap.String("Description", e.Description))
-		}
+		nerf.Cfg.Logger.Info("Authorized", zap.String("Login", nerf.Cfg.Login))
```
Comment (benner) [inline cmd/nerf/main.go]: Same here

File: nerf.go [hunk 6]
```diff
@@ -66,9 +65,7 @@ func (s *Server) Ping(ctx context.Context, in *PingRequest) (*PingResponse, erro
 		return nil, fmt.Errorf("Failed gRPC ping request")
 	}
 
-	if Cfg.Verbose {
-		Cfg.Logger.Info("Got ping request", zap.String("Login", *in.Login))
-	}
+	Cfg.Logger.Info("Got ping request", zap.String("Login", *in.Login))
```
Comment (benner) [inline nerf.go]: Debug

**Comment by benner** [comment]
Suggest to change all stuff who was under 'verbose' to .Debug()

File: cmd/nerf/main.go (hunk 3 above)
Comment (ton31337) [reply cmd/nerf/main.go:108]: I would say this is kinda more informational, not really debug. But I can change.

File: cmd/nerf/main.go (hunk 1 above)
Comment (ton31337) [reply cmd/nerf/main.go:21]: Fixed.

File: cmd/nerf/main.go (hunk 3 above)
Comment (ton31337) [reply cmd/nerf/main.go:108]: Fixed.

File: cmd/nerf/main.go (hunk 4 above)
Comment (ton31337) [reply cmd/nerf/main.go:111]: Fixed.

**Review by benner — APPROVED** [review]

**Pull request merged**